package-lock.json

```diff
@@ -7236,9 +7236,9 @@
             "dev": true
         },
         "node_modules/xml2js": {
-            "version": "0.5.0",
-            "resolved": "https://registry.npmjs.org/xml2js/-/xml2js-0.5.0.tgz",
-            "integrity": "sha512-drPFnkQJik/O+uPKpqSgr22mpuFHqKdbS835iAQrUC73L2F5WkboIRd63ai/2Yg6I1jzifPFKH2NTK+cfglkIA==",
+            "version": "0.4.23",
```

Comment:
Something seems off with this, why would the version go down

Reply:
Good catch, not sure - let me try regenerating the file again

Reply:
Make sure you're using node 18

Reply:
We can merge that first and I'll rebase on top of it

Reply:
This looks less weird now, but still weird that the versions are going down

Reply:
Yeah I went ahead and fixed it manually - I have no idea why that's happening, I'm using node `v18.17.1` - maybe something with my OS env?